src/08-accessing.js

```diff
@@ -171,7 +211,21 @@ function getFullAddress(person) {}
       numbers: [7185550921, 7185558611],
     };
  */
-function getFlatObject(person) {}
+function getFlatObject(person) {
+  const Diandre = {
```

Comment:
This isn't entirely hardcoded but there is a way more dynamic way to create the flat object. How can you use a for in loop to dynamically create the flat object?